src/main/java/com/checkout/payments/ProcessingSettings.java

```diff
@@ -35,4 +53,33 @@ public final class ProcessingSettings {
     @SerializedName("receipt_id")
     private String receiptId;
 
+    @SerializedName("terminal_type")
+    private TerminalType terminalType;
+
+    @SerializedName("os_type")
+    private OsType osType;
+
+    @SerializedName("invoice_id")
+    private String invoiceId;
+
+    @SerializedName("brand_name")
+    private String brandName;
+
+    private String locale;
+
+    @SerializedName("shipping_preference")
+    private ShippingPreference shippingPreference;
+
+    @SerializedName("user_action")
+    private UserAction userAction;
+
+    @SerializedName("set_transaction_context")
+    private List<Map<String, String>> setTransactionContext;
+
+    @SerializedName("airline_data")
+    private List<AirlineData> airlineData;
+
+    //Previous
```

Comment:
I think this comment should be referring to `Four` or `Default` as this is for the legacy version, right?